sdk/python/feast/errors.py

```diff
@@ -509,3 +509,25 @@ def grpc_status_code(self) -> GrpcStatusCode:
 
     def http_status_code(self) -> int:
         return HttpStatusCode.HTTP_403_FORBIDDEN
+
+
+class TagKeyAlreadyExists(Exception):
```

Comment:
All Feast errors should extend `FeastError` to be properly transferred from a server to a client proxy